lang/llvm-17/Portfile

```diff
@@ -29,9 +29,9 @@ version                 ${llvm_version}.0.6
 name                    llvm-${llvm_version}
 revision                1
 subport                 mlir-${llvm_version}  { revision [ expr ${revision} + 0 ] }
-subport                 clang-${llvm_version} { revision [ expr ${revision} + 2 ] }
-subport                 lldb-${llvm_version}  { revision [ expr ${revision} + 1 ] }
-subport                 flang-${llvm_version} { revision [ expr ${revision} + 1 ] }
+subport                 clang-${llvm_version} { revision [ expr ${revision} + 4 ] }
+subport                 lldb-${llvm_version}  { revision [ expr ${revision} + 0 ] }
+subport                 flang-${llvm_version} { revision [ expr ${revision} + 0 ] }
```

Comment:
I think the revisions got changed in the rebase. Only Clang needs a bump, right?